dependencies.gradle

```diff
@@ -1,5 +1,5 @@
 ext.versions = [
-        accompanist                 : '0.30.1',
+        accompanist                 : '0.32.0',
```

Comment:
Can probably avoid updating now with just the animated `NavHost` replacements

Reply:
While I could... This version is technically incompatible with compose 1.5. So updating seems like the right thing to do.